tests/rptest/tests/cloud_storage_usage_test.py

```diff
@@ -121,8 +121,8 @@ def check():
 
     def _test_epilogue(self):
         # Assert tht retention was active
-        self.redpanda.metric_sum(
-            "redpanda_cloud_storage_deleted_segments",
+        assert self.redpanda.metric_sum(
+            "redpanda_cloud_storage_deleted_segments_total",
```

Comment:
nice. a while back i added a parameter to `metric_sum` called `expect_metric` which when set to True will cause the function to throw if the metric isn't observed in the output. It's possible it should be the default.